pytest_xvfb.py

```diff
@@ -132,6 +133,12 @@ def pytest_addoption(parser):
 def pytest_configure(config):
     if config.getoption('--no-xvfb') or not xvfb_available():
         config.xvfb = None
+        if (sys.platform.startswith('linux')
+                and 'DISPLAY' in os.environ
+                    and not config.getoption('--no-xvfb')):
+                        print(
+                            'pytest-xvfb could not find Xvfb. '
+                            'You can install it to avoid windows from opening.')
```

Comment:
The example was a bit vague as it did not contain multiple conditional statements. There are also 8 spaces indenting the conditionals in your example, are you sure you don't want 4?

The latter statement makes more sense, the former is the sentence you gave when we first discussed the warning message.